packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -75,6 +75,7 @@ export const registerMgtPersonComponent = () => {
  * @class MgtPerson
  * @extends {MgtTemplatedComponent}
  *
+ * @fires {CustomEvent<null>} updated - Fired when the component is updated
```

Comment:
Add this to the https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/main/packages/mgt-element/src/components/baseTaskComponent.ts file then re-run `yarn build`. This will generate the props for all components that fire the event. Then add this comment on the components whose generated files have been updated for consistent documentation.

```suggestion
 * @fires {CustomEvent<undefined>} updated - Fired when the component is updated
```

Reply:
What are you trying to accomplish with my software?